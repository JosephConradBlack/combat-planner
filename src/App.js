import React, { Component } from 'react';
import { ThemeProvider } from 'styled-components';

import theme from './theme';
import CombatPlannerContainer from './containers/CombatPlannerContainer';

class App extends Component {
  render() {
    return (
      <div>
        <ThemeProvider theme={theme}>
          <CombatPlannerContainer />
        </ThemeProvider>
      </div>
    );
  }
}

export default App;
