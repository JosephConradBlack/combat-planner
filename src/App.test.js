import React from 'react';
import App from './App';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

it('renders without crashing', () => {
  const component = shallow(<App />);
  expect(toJson(component)).toMatchSnapshot();
  expect(component.instance()).toBeInstanceOf(App);
});
