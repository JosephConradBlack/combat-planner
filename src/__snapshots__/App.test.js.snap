// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <ThemeProvider
    theme={
      Object {
        "h1": Object {
          "fontSize": "4em",
          "fontWeight": 300,
          "letterSpacing": "0.06em",
          "subheader": Object {
            "fontSize": "1em",
            "fontWeight": "lighter",
          },
        },
      }
    }
  >
    <CombatPlannerContainer />
  </ThemeProvider>
</div>
`;
